doc/Internals.md

```diff
@@ -29,13 +29,32 @@ A WDL workflow is compiled into an equivalent DNAnexus workflow, enabling runnin
 * Each scatters or conditional block is compiled into workflow stages, plus an auxiliary applet
     - Nested scatters/conditionals may result in nested workflows 
 
-The mapping from WDL to native DNAnexus objects requires overcomming two key obstacles:
+The mapping from WDL to native DNAnexus objects requires overcoming two key obstacles:
 
 * We wish to avoid creating a controlling applet that would run and manage a WDL workflow. Such an
 applet might get killed due to temporary resource shortage, causing an
 expensive workflow to fail. 
 * We want to minimize the context that needs to be kept around for the WDL workflow, because it limits job manager scalability.
 
+### Examining compilation output
+
+After compiling your workflow to a project, you will see the compiled workflows and applets in
+the project like this:
+
+```
+my_task_1
```

Comment:
are there any hidden applets? frags?

Reply:
The example reflects the visible applets according to the current implementation, so not anything that would be compiled as hidden.

Reply:
@emiloslavsky   I left myself a note on https://jira.internal.dnanexus.com/browse/APPS-623 to revisit this doc while implementing the bundledDepends task, and ensure the explanation about visibility is improved / up to date then.